src/fides/api/schemas/connection_configuration/connection_secrets.py

```diff
@@ -29,24 +24,21 @@ def __init_subclass__(cls: BaseModel, **kwargs: Any):  # type: ignore
     def required_components_supplied(  # type: ignore
         cls: ConnectionConfigSecretsSchema, values: Dict[str, Any]
     ) -> Dict[str, Any]:
-        """Validate that the minimum required components have been supplied.
-
-        Connection configurations either 1) need the entire URL
-        *OR* 2) all of the required components to *build* that URL."""
-        min_fields_present = values.get("url") or all(
+        """Validate that the minimum required components have been supplied."""
+        min_fields_present = all(
             values.get(component) for component in cls._required_components
         )
         if not min_fields_present:
             raise ValueError(
-                f"{cls.__name__} must be supplied a 'url' or all of: {cls._required_components}."  # type: ignore
+                f"{cls.__name__} must be supplied all of: {cls._required_components}."  # type: ignore
             )
 
         return values
 
     class Config:
         """Only permit selected secret fields to be stored."""
 
-        extra = Extra.forbid
+        extra = Extra.ignore
```

Comment:
We don't want validation to fail for connectors with urls in the secrets. Just ignore extra fields instead of throwing a validation error.